doc/manual/general.rst

```diff
@@ -32,13 +32,18 @@ Introduction
 
 The SoundScape Renderer (SSR) is a software framework for real-time
 spatial audio reproduction running under GNU/Linux, Mac OS X and
-possibly some other UNIX variants. The current implementation provides
-Wave Field Synthesis (WFS), binaural (HRTF-based) reproduction, binaural
-room (re-)synthesis (BRTF-based reproduction), head-tracked binaural
-playback, Ambisonics Amplitude Panning (AAP), and Vector Base Amplitude
-Panning (VBAP). There are also the slightly exotic Generic Renderer. For
-each rendering algorithm there is a separate executable file. For more
-details see section :ref:`Renderers <renderers>`.
+possibly some other UNIX variants. The MS Windows version is experimental. 
+
+The current implementation provides:
+
+- Wave Field Synthesis (WFS)
+- binaural (HRTF-based) reproduction
+- binaural room (re-)synthesis (BRTF-based reproduction)
+- head-tracked binaural playback
```

Comment:
I think this renderer ceased to be, right?

Reply:
Oh, right, I'll remove this...